recipes/cpptango/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "cppTango" %}
+{% set version = "9.3.4" %}
+{% set sha256 = "50828ae9f523c8b2eeb546b38138b0d0882fcd3050e952d93a66a594ed178e72" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tango-controls/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports :
+    - {{ pin_subpackage('cpptango', min_pin='x.x', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('cxx') }}
+    - libtool
+    - pkg-config
+  host:
+    - omniorb
+    - cppzmq
+    - zeromq
+    - tango-idl
+  run:
+    - {{ pin_compatible('omniorb', min_pin='x.x', max_pin='x.x') }}
```

It is really okay to run with an older version than was built? i.e. building with 1.2.3 but running with 1.2.0?
```suggestion
    - {{ pin_compatible('omniorb', max_pin='x.x') }}
```

---

I think it should be ok since these versions are supposed to be binary compatible.

---

I think it should be ok since these versions are supposed to be binary compatible.

---

If they add new symbols it would be forwards compatible with new x.x releases but not backwards compatible with older x.x releases. Most projects would still class this as being "binary compatible" but if you're sure these two packages will also not add new symbols it can be left as-is.

---

Indeed, new symbols could be added. So your comment is pertinent and I think your suggestion sounds like the safest way to go.
Thanks!